### PR TITLE
WEB: Remove unmaintained projects from Ecosystem

### PR DESCRIPTION
**Pull request by datapythonista**
Lots of projects in our Ecosystem page seem to be obsolete and irrelevant now. There are few more that could probably be removed to help users find the relevant ones faster and easier, but I'm just removing the ones that didn't have any commit for at least two years.

### CODE REVIEW

File: web/pandas/community/ecosystem.md [hunk 1]
```diff
@@ -559,12 +482,6 @@ Arrays](https://awkward-array.org/) inside pandas' Series and
 DataFrame. It also provides an accessor for using awkward functions
 on Series that are of awkward type.
 
-### [cyberpandas](https://cyberpandas.readthedocs.io/en/latest)
```
[the remark this answers is not in the document]
Comment (datapythonista) [reply web/pandas/community/ecosystem.md:562]: Sure. I'll have a look to see if it still works with the latest pandas. I checked that not only didn't have a commit in many years, but it also didn't have any bug report, so I guess it's not used much.

[the remark this answers is not in the document]
Comment (MarcoGorelli) [reply web/pandas/community/ecosystem.md:562]: I don't think it's used at all https://pypistats.org/packages/cyberpandas

[the remark this answers is not in the document]
Comment (datapythonista) [reply web/pandas/community/ecosystem.md:562]: @jreback I tried, and installing cyberpandas from conda-forge gives an installation error (I guess the recipe is too old) and installing with pip works but fails on import:

```
>>> import pandas
>>> from cyberpandas import IPArray
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/mgarcia/.mambaforge/envs/cyberpandas/lib/python3.12/site-packages/cyberpandas/__init__.py", line 10, in <module>
    from .mac_array import MACType, MACArray
  File "/home/mgarcia/.mambaforge/envs/cyberpandas/lib/python3.12/site-packages/cyberpandas/mac_array.py", line 1, in <module>
    from collections import Iterable
ImportError: cannot import name 'Iterable' from 'collections' (/home/mgarcia/.mambaforge/envs/cyberpandas/lib/python3.12/collections/__init__.py)
```

I think it's worth removing from the Ecosystem.

CC: @TomAugspurger

[the remark this answers is not in the document]
Comment (TomAugspurger) [reply web/pandas/community/ecosystem.md:562]: OK to remove IMO. I don't think it's maintained these days.

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @datapythonista

**Pull request merged**